src/pages/Details/components/SubscribeOptions.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import './SubscribeOptions.scss';
+
+const SubscribeOptions = () => {
+  const [selected, setSelected] = useState('');
+  const selectList = [
+    '[필수] 배송주기를 선택해 주세요.',
+    '1주일',
+    '1개월',
+    '3개월',
+  ];
+  const handleCycleClick = e => {
+    return e.target.selectedIndex === 0
```

Comment:
```suggestion
   e.target.selectedIndex === 0
```
여기도 반환값이 없기 때문에 return 을 안해주셔도 됩니다!